src/components/PopupMenu.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
```

Comment:
-2018

Reply:
Actually 2019 right? We need to update them all, but I did this one.